pkg/kubelet/pod/mirror_client_test.go

```diff
@@ -54,3 +57,76 @@ func TestParsePodFullName(t *testing.T) {
 		}
 	}
 }
+
+func TestIsStaticPod(t *testing.T) {
+	testcases := []struct {
+		pod    *v1.Pod
+		expect bool
```

Comment:
rename "expect" -> "expectStaticPod"?

Reply:
debatable. I feel together with the 'actual' and function name, "expect" in this context should be self-explanatory in meaning.